### ISSUE TITLE
Conditional does not appear to work with platform_system

### ISSUE THREAD

**Issue by sagshome** [wemake-services/coverage-conditional-plugin#189]
Here is my rule:

[coverage_conditional_plugin]
rules =
    "platform_system == 'Windows'": escape_win

Here is my code:
        if remove:
            try:
                self.de_register(silent=True)
                os.unlink(self.path)
            except OSError as error:   # pragma: escape_win
                logger.debug('%s could not be removed (%s)', self.path, error)

This is my environment:

get_env_info()['platform_system']
'Windows'

**Comment by inorton** [comment]
I just found out that the match means "no cover if" eg:
```
rules =
    "platform_system == 'Windows'":  windows
```
means: "no cover if on windows"

**Comment by sagshome** [comment]
> I just found out that the match means "no cover if" eg:
> 
> ```
> rules =
>     "platform_system == 'Windows'":  windows
> ```
> 
> means: "no cover if on windows"

Agreed,   that is why I named it escape_win,   I don't want that code fragment counted if I am on Windows.

**Comment by sagshome** [comment]
Just as a follow up.   I thought I would upgrade to see if that solved my issue.   Still no luck,  however - if I move my .coveragerc file to the root of my project it works.     I guess I will live with that for now.   I will change the title of this issue
